pay-api/src/pay_api/services/payment.py

```diff
@@ -161,11 +175,21 @@ def save(self):
 
     def asdict(self):
         """Return the payment as a python dict."""
+        invoices = []
+        for invoice in self._invoices:
+            if Invoice.populate(invoice).invoice_status_code != Status.CANCELLED.value:
+                invoices.append(Invoice.populate(invoice).asdict())
+
```

Comment:
It would be better to assign Invoice.populate.. to a variable..

Reply:
updated.